operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -1284,5 +763,516 @@ spec:
-      version: v1alpha1
+      version: v1beta2
+  description: >
+    **Red Hat AMQ Streams** is a massively scalable, distributed, and high performance data streaming platform based on the Apache Kafka® project.
+    AMQ Streams provides an event streaming backbone that allows microservices and other application components to exchange data with extremely high throughput and low latency.
+
+    **The core capabilities include:**
+
+    * A pub/sub messaging model, similar to a traditional enterprise messaging system, in which application components publish and consume events to/from an ordered stream
+
+    * The long term, fault-tolerant storage of events
+
+    * The ability for a consumer to replay streams of events
+
+    * The ability to partition topics for horizontal scalability
+
+    ### CRD Upgrades
+
+    **!!! IMPORTANT !!!** 
+    This release introduces new API version `v1beta2` to all Strimzi custom resources. 
+    This is a preparation for migration to `apiextensions/v1` which is needed because Kubernetes 1.22 will remove support for `apiextensions/v1beta1`. 
+    Migration to `v1beta2` needs to be completed for all Strimzi CRDs and CRs after the upgrade to 0.22 is done and before upgrading to Strimzi 0.23 which will support only Strimzi `v1beta2` APIs and `apiextensions/v1` CRDs.
+    For more details about the CRD upgrades, see the [documentation](https://strimzi.io/docs/operators/0.22.1/deploying.html#assembly-upgrade-resources-str).
+
+    ### What is new in AMQ Streams 1.7
+
+    * Add v1beta2 version for all resources. v1beta2 removes all deprecated fields.
+
+    * Add annotations that enable the operator to restart Kafka Connect connectors or tasks. The annotations can be applied to the KafkaConnector and the KafkaMirrorMaker2 custom resources.
+
+    * Add additional configuration options for the Kaniko executor used by the Kafka Connect Build on Kubernetes
+
+    * Add support for JMX options configuration of all Kafka Connect (KC, KC2SI, MM2)
+
+    * Update Strimzi Kafka OAuth to version 0.7 and add support for new features (OAuth over SASL PLAIN, Checking token audience, Validating tokens using JSONPath)
+
+    * Configure external logging ConfigMap name and key.
+
+    * Add support for configuring labels and annotations in ClusterRoleBindings created as part of Kafka and Kafka Connect clusters
+
+    * Add support for Ingress v1 in Kubernetes 1.19 and newer
+
+    * Add support for Kafka 2.6.1
+
+    * List topics used by a Kafka Connect connector in the .status section of the KafkaConnector custom resource
+
+    * Bump Cruise Control to v2.5.37 for Kafka 2.7 support. Note this new version of Cruise Control uses Log4j 2 and is supported by dynamic logging configuration (where logging properties are defined in a ConfigMap). However, existing Log4j configurations must be updated to Log4j 2 configurations.
+
+    * Support pausing reconciliation of CR with annotation strimzi.io/pause-reconciliation
```

Comment:
Maybe we should remove this whole section. It has two problems:
* It covers things AMQ Streams do nto support (Ingress, Kafka 2.6.1, etc.)
* It covers only 0.22. Not 0.21 features. But AMQ Streams 1.7 has to include both. So if we want to keep this, we should extend this with the new features from 0.21 as well.

Reply:
I removed this altogether to avoid including incorrect information.